File: Project_Scala/src/main/scala/Node.scala
class Node(var symbol:String, var children:List[ParserTree], override val start:Int, override val end:Int) extends ParserTree(start,end) {
  def print(prefix:String, continuation:String):Unit = {
    println(prefix + symbol)
    for(i <- 0 until children.size-1) {
      children(i).print(continuation+ "├──", continuation + "│  ")
    }
    children.last.print(continuation + "└──", continuation+ "   ")
  }
}

File: Project_Python/src/Node.py
class Node(ParserTree):
    def __init__(self, symbol, children, start, end):
        self.symbol=symbol
        self.children=children
        self.start=start
        self.end=end

    def print(self, string, prefix, continuation):
        string.append(prefix).append(symbol).append("\n")
        i=0
        while i <  self.children.len()-1 :
            self.children[i].print(string, continuation+"├──", continuation+"│  ")
        children[(children.len() - 1).print(string, continuation + "└──", continuation + "   ")]